tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -99,3 +99,39 @@ describe('Test GitRepo Bundle name validation and max character trimming behavio
     }
   )
 });
+
+describe('Test resource behavior after deleting GitRepo using keepResources option', { tags: '@p1'}, () => {
+  const repoName = "local-cluster-fleet-71"
+  const branch = "master"
+  const path = "qa-test-apps/nginx-app"
+  const repoUrl = "https://github.com/rancher/fleet-test-data/"
+  const appNamespace = 'nginx-keep'
+  const appName = 'nginx-keep'
+  const keepResourceData: testData[] = [
+    { qase_id: 69,
+      keepResources: 'true',
```

Comment:
Just a comment / opinion (not a stopper), but normally when using Booleans I guess better not use string typed for true/false

Reply:
I will update it.